lib/airbrake-ruby/time_truncate.rb

```diff
@@ -9,7 +9,8 @@ module TimeTruncate
     # @param [Time] time
     # @return [String]
     def self.utc_truncate_minutes(time)
-      tm = time.getutc
+      # time does not have getutc method instead of Time on Rails 7.0.0
```

Comment:
IMO, it might be clearer to remove the "instead of Time" bit. eg:
```suggestion
      # In Rails 7.0.0, time is a float and does not have the getutc method
```

Reply:
even better : `..time is a Float..`